spring-cloud-gcp-samples/spring-cloud-gcp-security-iap-sample/README.adoc

```diff
@@ -0,0 +1,7 @@
+TODO
```

Comment:
TODO?

Reply:
I've been retelling this story in English for years. Finally, I can cite it "in the original Klingon" (well, except when I first encountered it, the difficulty was with getting a painting past government censors, and not a strict editor): https://koppel.pro/diary/sobacka-favorskogo-2443